### ISSUE TITLE
Cartridge: atom and bond properties not serialized

### ISSUE THREAD

**Issue by greglandrum** [rdkit/rdkit#6310]
**Describe the bug**
No atom or bond properties are preserved when a molecule is serialized.
This leads to problems with atom labels, as shown in the demo, but will also have knock-on effects since we use those properties for a bunch of different stuff.

```
chembl_32=# select mol_to_cxsmiles(mol_from_smiles('F* |$;HAR_p$|'));
       mol_to_cxsmiles        
------------------------------
 *F |atomProp:0.dummyLabel.*|
(1 row)
```

**Configuration (please complete the following information):**
 - RDKit version: master and releases
 - OS: all

**Comment by greglandrum** [comment]
Among other things, this is also not allowing the generic searches added in #6269 to work when molecules come in via cxsmiles

**Comment by greglandrum** [comment]
I believe this was fixed in #6332
